easybuild/easyconfigs/g/GPyTorch/GPyTorch-1.9.1-foss-2021a-CUDA-11.3.1.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'PythonBundle'
+
+name = 'GPyTorch'
+version = '1.9.1'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://gpytorch.ai'
+description = "GPyTorch is a Gaussian process library implemented using PyTorch."
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('CUDA', '11.3.1', '', SYSTEM),
+    ('Python', '3.9.5'),
+    ('PyTorch', '1.12.1', versionsuffix),
+    ('scikit-learn', '0.24.2'),
+]
+
+download_dep_fail = True
+use_pip = True
+
+exts_list = [
+    ('linear-operator', '0.3.0', {
+        'source_tmpl': 'linear_operator-%(version)s.tar.gz',
+        'checksum': ['84bf572631a7e1576de6920d81600ca0fedcf6bda2f29dbaf440d6e72ce6abab'],
```

Comment:
This is a typo: the correct keyword is `checksums` (multiple). That's why CI complains that there is still no checksum found.